Gemfile.lock

```diff
@@ -112,6 +112,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-22
+  x86_64-darwin-21
```

Comment:
This was added by `bundle install`. Probably good to keep it as this build platform will be shared by others as well.

Reply:
Sounds like a good idea, thanks for highlighting it with a comment :)